data/models/canteen.py

```diff
@@ -564,7 +564,7 @@ def save(self, force_insert=False, force_update=False, using=None, update_fields
             self.logo = optimize_image(self.logo, self.logo.name, max_image_size)
         if self.department:
             self.region = self._get_region()
-        super().save(force_insert, force_update, using, update_fields)
+        super().save(force_insert=force_insert, force_update=force_update, using=using, update_fields=update_fields)
```

Comment:
sauf si on utilise les kwargs (force_insert, force_update, using, update_fields), ca peut être simplifié en
```suggestion
        super().save(*args, **kwargs)
```

Reply:
exemple ici
https://docs.djangoproject.com/en/4.2/ref/models/instances/